web/web.go

```diff
@@ -346,10 +347,15 @@ func New(logger log.Logger, o *Options) *Handler {
 		router = router.WithPrefix(o.RoutePrefix)
 	}
 
+	homePage := "/graph"
```

Comment:
Since this is only used in the `router.Get()` below, maybe pull the variable definition in there? Makes the limited scope clearer to the reader.